src/main/java/ro/developmentfactory/thedrones/exception/LoggingAspect.java

```diff
@@ -0,0 +1,20 @@
+package ro.developmentfactory.thedrones.exception;
+
+import org.aspectj.lang.annotation.Aspect;
+import org.aspectj.lang.annotation.Before;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.stereotype.Component;
+
+@Aspect
+@Component
+public class LoggingAspect {
```

Comment:
it looks like you just write the line `"A method in service layer is about to be executed."` every time you call a service method. do you need it? it's probably not very informative and you already log a lot